larray/core/group.py

```diff
@@ -1275,6 +1275,39 @@ def containing(self, substring):
             substring = substring.eval()
         return LGroup([v for v in self.eval() if substring in v], axis=self.axis)
 
+    def to_hdf(self, filepath, key, axis_key, **kwargs):
+        """
+        Writes group to a HDF file.
+
+        A HDF file can contain multiple groups.
+        The 'key' parameter is a unique identifier for the group.
+
+        Parameters
+        ----------
+        filepath : str
+            Path where the hdf file has to be written.
+        key : str or Group
+            Name of the group within the HDF file.
+        axis_key : str
+            key of the associated axis in the HDF file.
+        **kwargs
+
+        Examples
+        --------
+        >>> from larray import Axis
+        >>> a = Axis("a=a0..a2")
+        >>> a.to_hdf('test.h5', 'a')
+        >>> a01 = a['a0,a1'] >> 'a01'
+        >>> a01.to_hdf('test.h5', 'a01', 'a')   # doctest: +SKIP
+
+        Save group in a specific HDF group
+
+        >>> a.to_hdf('test.h5', 'groups/a', 'a')   # doctest: +SKIP
+        """
+        key = _translate_key_hdf(key)
+        s = pd.Series(data=self.eval(), name='{}@{}'.format(self.name, axis_key))
```

Comment:
Ideally we should test that the axis is in the HDF file, but that will probably going to cause an ordering problem (there is currently no guarantee that the group will be saved after the axis), so testing that the correct axis exists in the session seems like a good proxy. It would still be possible to save a group without its axis, but that would be much harder.